webpack.config.js

```diff
@@ -34,6 +34,10 @@ var plugins = [
     {
       from: 'static',
       transform: (content, path) => {
+        if (isDev) {
```

Comment:
Question - was this a workaround that is meant to be checked in? FYI I have the same on my branch so just interested

Reply:
I can remove this now, as it's now pushed to master and will merge in when I prepare this build for staging.